data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/GetNewBlindedBlock.java

```diff
@@ -170,9 +155,9 @@ private static EndpointMetadata getEndpointMetaData(
                 + "will be added to as forks progress.\n\n"
                 + "Pre-Bellatrix, this endpoint will return a `BeaconBlock`.")
         .tags(TAG_VALIDATOR, TAG_VALIDATOR_REQUIRED, TAG_EXPERIMENTAL)
-        .pathParam(PARAM_SLOT)
-        .queryParamRequired(PARAM_RANDAO)
-        .queryParam(PARAM_GRAFFITI)
+        .pathParam(SLOT_PATH_PARAMETER)
```

Comment:
so if you get rid of the path_param, this is just SLOT_PARAMETER.withDescription("...")